SquiggleNet/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# SquiggleNet requires Python >= 3.7
+FROM --platform=linux/amd64 python:3.11 AS build
+
+WORKDIR /app
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends build-essential gcc
+
+# PyTorch CPU version (or should the container support GPU?)
+RUN pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+
+RUN pip3 install scipy numpy click ont-fast5-api
+
+RUN pip3 install typing-extensions --upgrade
+
+RUN git clone https://github.com/welch-lab/SquiggleNet
```

Comment:
We should use Git submodules to specify a specific version. This approach allows us to always build the same image and prevents unexpected breaks when a new version is released on SquiggleNet Repository. We can test new versions before upgrading, ensuring stability.

Reply:
By the way, this is completely optional; we can skip this if needed. The image looks great, well done! 😄